hoomd/MPIConfiguration.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 // ensure that HOOMDMath.h is the first thing included
+#include "ClockSource.h"
```

Comment:
This works on Linux, but I'm not sure if it's provided by all systems. Verify please. Is there a way we can use a standard library or reason not to?

Reply:
I wasn't aware it was already used. Clearly it's no issue to include it another place.